client/flutter/lib/pages/onboarding/legal_landing_page.dart

```diff
@@ -36,12 +36,14 @@ class LegalLandingPage extends StatelessWidget {
                   SizedBox(height: 8),
                   Semantics(
                     container: true,
-                    child: ThemedText(
-                      S.of(context).legalLandingPageTitle,
-                      variant: TypographyVariant.body,
-                      style: TextStyle(
-                        color: Constants.primaryColor,
-                        fontSize: 18,
+                    child: Center(
+                      child: ThemedText(
+                        S.of(context).legalLandingPageTitle,
+                        variant: TypographyVariant.body,
```

Comment:
```suggestion
                       textAlign: TextAlign.center,
                        variant: TypographyVariant.body,
```

Reply:
@theswerd I have made changes kindly go through that once.